earn/src/pages/boost/ManageBoostPage.tsx

```diff
@@ -171,8 +171,11 @@ export default function ManageBoostPage() {
       {!isLoading && (
         <Container>
           <BoostCard info={cardInfo} uniqueId={nftTokenId} isDisplayOnly={true} />
-          <div className='flex-grow'>
+          <div className='flex-grow flex-col'>
```

Comment:
Remove `flex-col`, it doesn't do anything here since this is a block element.

Reply:
You can do this when you change styling